grails-app/views/product/_productSynonyms.gsp

```diff
@@ -39,8 +39,15 @@
                             ${g.message(code: 'enum.SynonymTypeCode.' + synonym.synonymTypeCode)}
                         </td>
                         <td class="middle">
+                            <a href="javascript:void(0);" class="button btn-show-dialog"  data-width="900"
+                               data-title="${warehouse.message(code:'inventoryLevel.edit.label', default: 'Edit synonymss')}"
```

Comment:
should the code stay `inventoryLevel...`? I can also see a typo - "Edit synonyms**s**"

Reply:
I didn't notice that, thanks